src/tests.rs

```diff
@@ -223,9 +220,7 @@ fn test_rust_function() {
     ).unwrap();
 
     let lua_function = globals.get::<_, Function>("lua_function").unwrap();
-    let rust_function = lua.create_function(|lua, _| {
-        lua.pack("hello")
-    });
+    let rust_function = lua.create_function(|_, _: ()| Ok("hello"));
```

Comment:
Oh yeah!  I forgot you mentioned that, that's a great idea.  I guess it's allowed because it's an irrefutable pattern, and that's allowed as lambda arguments?  I have never actually used that before!